README.md

```diff
@@ -0,0 +1,188 @@
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Enumerable <a name="about-project"></a>
+
+**Enumerable** In this project, we utilize the Ruby language to write a class List and module enumerable.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+<summary>RUBY</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Ruby Terminal</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **The project implements #all**
+- **The project implements #any**
+- **The project implements #filter**
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+ cd my-folder
+ https://github.com/Jawad-Nazari/Enumerables.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd enumerable
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Run ruby my_enumerable.rb
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+```sh
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Ernest Korir**
+
+- GitHub: [@ernestkorir](https://github.com/ernestkorir)
+- LinkedIn: [Ernest korir](https://www.linkedin.com/in/ernest-korir/)
+
+👤 **Jawad Nazari**
+
+- GitHub: [@Jawad-Nazari](https://github.com/Jawad-Nazari)
+- LinkedIn: [Jawad-Nazari](https://www.linkedin.com/in/jawad-nazari/)
+- Twitter: [@nazari_jawad58](https://twitter.com/nazari_jawad58)
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Building_a_GUI_version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Jawad-Nazari/Enumerables.git).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] This link is misleading, I was redirecte to a wrong page. Kindly justify it. ℹ️ **Perhaps you meant `https://github.com/Jawad-Nazari/Enumerables/issues`**.